packages/backend-core/src/events/publishers/group.ts

```diff
@@ -25,6 +26,7 @@ async function created(group: UserGroup, timestamp?: number) {
 async function updated(group: UserGroup) {
   const properties: GroupUpdatedEvent = {
     groupId: group._id as string,
+    viaScim: context.isScimCall(),
```

Comment:
personal pref
```suggestion
    isScim: context.isScim(),
```

Reply:
That was my first approach, but we could run into some case that we are deleting a SCIM user manually. The field here should be false, even if the user is scim sync. And it might be confusing

Reply:
The above suggestion was just for the naming, I don't think the settings on the user would impact it